src/open_inwoner/openzaak/models.py

```diff
@@ -623,6 +636,11 @@ def informatieobjecttype_uuid(self):
     def __str__(self):
         return f"{self.omschrijving} [{self.zaaktype_config.catalogus.base_url}]"
 
+    def natural_key(self):
+        return (self.informatieobjecttype_url,) + self.zaaktype_config.natural_key()
```

Comment:
```suggestion
        return (self.informatieobjecttype_url, self.zaaktype_config.natural_key())
```
Same below a few times

Reply:
That's fine, thanks for the explanation!